sensor_msgs/Image left_image
sensor_msgs/Image right_image
sensor_msgs/CameraInfo left_camera_info
sensor_msgs/CameraInfo right_camera_info

---

stereo_msgs/DisparityImage DisparityValue
std_msgs/Float32 cx_l
std_msgs/Float32 cx_r
std_msgs/Float32 cy_l
std_msgs/Float32 cy_r
